docker-experimental-swarm-cluster/azuredeploy.json

```diff
@@ -0,0 +1,254 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "allowedValues": [ "East US", "West US", "West Europe", "East Asia", "South East Asia" ],
+            "metadata": {
+                "description": "This is the location where the cluster will be deployed"
+            }
+        },
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the storage account where the virtual hard disks of the VMs are going to be stored"
+            }
+        },
+        "virtualNetworkName": {
```

Comment:
This could become a variable to reduce the number of parameters a user has to enter